interface/forms/clinical_instructions/new.php

```diff
@@ -51,20 +51,21 @@
         <?php call_required_libraries(['bootstrap']); ?>
     </head>
     <body class="body_top">
-        <h2><?php echo xlt('Clinical Instructions Form'); ?></h2>
-        <h3><?php echo xlt('Instructions').':'; ?></h3>
+        <center>
+            <h2><?php echo xlt('Clinical Instructions Form'); ?></h2>
+            <h3><?php echo xlt('Instructions').':'; ?></h3>
 
-        <!-- Begin form -->
-        <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/clinical_instructions/save.php?id=" . attr($formid) . "'>\n"; ?>
-            <div class="container" style="width: 500px; float: left;">
-                <textarea class="form-control" name="instruction" id ="instruction" rows=3>
-                    <?php echo text($check_res['instruction']); ?>
-                </textarea>
-                <br/>
-                <input type='submit' value='<?php echo xla('Save'); ?>' class="button-css cp-submit" style="float: right">&nbsp;
-            </div>
-
-        </form>
+            <!-- Begin form -->
+            <?php echo "<form method='post' name='my_form' " . "action='$rootdir/forms/clinical_instructions/save.php?id=" . attr($formid) . "'>\n"; ?>
+                <div class='form-group'>
+                    <textarea class="form-control" name="instruction" id ="instruction" rows=3 style='resize: vertical;'>
+                        <?php echo text($check_res['instruction']); ?>
```

Comment:
Here the cursor starts at about 1/3 of the box can you please fix that (try clicking on the text area box)

Reply:
@muarachmann fixed. I know I'm supposed to be patient with task approval, but in light that it is the last 2 days to claim a task. Could you go to my task instance and approve it if this PR is good? https://codein.withgoogle.com/dashboard/task-instances/5694115258826752/